cloudinit/sources/DataSourceOpenStack.py

```diff
@@ -59,6 +59,13 @@ class DataSourceOpenStack(openstack.SourceMixin, sources.DataSource):
         }
     }
 
+    default_update_events = {
```

Comment:
I'd want to get a deb of this made and run an integration test on openstack using a snapshot containing this changeset to assert we aren't seeing significant boot speed/networking issues across reboot with this approach.